### PR TITLE
Standardize coffee recipes

### PR DESCRIPTION
**Pull request by Glamurio**
[QoL]
## About the PR 
This adds a /fresh variant to any recipes that require coffee, essentially allowing both freshly brewed coffee and regular coffee to be mixed to get the resulting drink.

## Why's this needed? 
The fact that "freshly brewed coffee" and "coffee" are two different chemicals is very confusing, especially to newer players playing bartender. It was also needlessly restrictive. This should allow for more freedom when creating drinks that contain coffee, without homogenizing the chemicals. (Though that may be a future step)

This is part of an ongoing series of changes I am planning to standardize coffee makers and espresso machines.

## Changelog 

```changelog
(u)Glamurio (Ryou)
(+)Standardized all coffee recipes to work with either regular coffee, or its freshly brewed variant.
```

### CODE REVIEW

**Comment by DisturbHerb** [comment]
this is probably more an indictment on recipe code itself than anything else :))

**Comment by ZeWaka** [comment]
please label your PRs, ty

**Comment by Glamurio** [comment]
> please label your PRs, ty

Sorry, pretty much always do, just forgot about Catering

**Review by Tarmunora — APPROVED** [review]

**Comment by Tarmunora** [comment]
merge conflicts

**Comment by Tarmunora** [comment]
why is this showing like 1500 lines added. I think you messed up resolving the merge conflict and now there are 2 of each recipe in the file

**Comment by Glamurio** [comment]
> why is this showing like 1500 lines added. I think you messed up resolving the merge conflict and now there are 2 of each recipe in the file

You're very right, god this indentation thing mid-PR really tripped me up, it should be fixed now

**Review by Tarmunora — CHANGES_REQUESTED** [review]
branch is still kinda fucked up

**Comment by Glamurio** [comment]
This was the stupidest PR ever, next time this happens I'll just scrap it and do it over
It should HOPEFULLY be fixed now

**Comment by Azrun** [comment]
I think the coffee things are gone now? 😕

**Comment by Tarmunora** [comment]
Yeah the only change in the branch rn is deleting a single line of whitespace

**Comment by Glamurio** [comment]
So good at fixing I undid the whole PR. Officially worst contributor of all time. I'll fix it when I have some time, sorry to have wasted your time, I feel really bad about this

File: code/modules/chemistry/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -1813,7 +1831,11 @@
 		result_amount = 3
 		mix_phrase = "You hear a faint quack from the solution along with a pungent stench."
 		mix_sound = 'sound/voice/farts/fart3.ogg'
-		drinkrecipe = TRUE
+		drinkrecipe = 1
```
Comment (Azrun) [inline code/modules/chemistry/Chemistry-Recipes.dm:1834]: `TRUE`

**Review by Azrun — CHANGES_REQUESTED** [review]

**Pull request merged**